docker-compose.yml

```diff
@@ -1,21 +1,6 @@
 version: '3'
 
 services:
-  dev: &dev
```

Comment:
can we keep the dev service but use the `term` service setup?

Reply:
you just want to call it dev instead of term?

Reply:
Yeah I think dev makes more sense. We can just stay zshell is our default dev terminal